SWLOR.Game.Server/Service/Spawn.cs

```diff
@@ -307,12 +307,18 @@ private static void RemoveQueuedSpawn(QueuedSpawn queuedSpawn)
 
         /// <summary>
         /// When a creature dies, its details need to be queued up for a respawn.
+        /// NOTE: plc_death and crea_death will not trigger if the object is 'killed'
```

Comment:
Just another place that needs attention after we add the script call to the DestroyObject override.